DXRMapFixups/DeusEx/Classes/DXRFixupIntro.uc

```diff
@@ -11,6 +11,11 @@ function PreFirstEntryMapFixes()
         foreach AllActors(class'#var(DeusExPrefix)Mover',dxm,'trap_door'){
             DestroyMover(dxm);
         }
+
+        if (!dxr.flags.IsZeroRando()) {
```

Comment:
Could check the flag for the Aug bots directly

Reply:
Yeah, that's better.